### ISSUE TITLE
IBM comment 2 on 2.2.6 Accessible Authentication

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag21#441]
The exception "not achievable due to legal requirements" is insufficient to accommodate an organization which emphasizes heightened scrutiny outside a legal framework. IBM is unaware of techniques that would allow this SC to be met which do not place heavy reliance on email (or similar delivery mechanisms) as a means of authentication. Essentially, any individual who achieves access to an individual's email account would be able to bypass authentication for any web application that uses an ID validated via that email account, as the current wording does not require the provision of a password or security question response to alter the user's authentication.

**Comment by alastc** [comment]
> any individual who achieves access to an individual's email account would be able to bypass authentication for any web application that uses an ID validated via that email account

I think that is the case now, without 2FA you can generally get into anything via access to the person's email.

However, I do think we need to show a secure way of logging in in the following scenarios:

- Where you are logging into your webmail (therefore can't rely on email reset).
- Where you need 2FA and not relaying information from one device to another (without SMS)

NB: Not using [SMS as it is on it's way out](https://www.theverge.com/2017/9/18/16328172/sms-two-factor-authentication-hack-password-bitcoin) as a 2FA method.

I cannot see how to achieve that at the moment without undermining the intent of the SC.

**Comment by joshueoconnor** [comment]
@mbgower @alastc Guys - has this comment been over taken by events? Please let me know so I can close if so.

**Comment by alastc** [comment]
I don't think the general "There is no way to do this in a 2FA scenario" has changed, and the text hasn't changed since this was opened.

I did post some ideas on how to scope it differently that would (I think) address this, but it wasn't taken up.

**Comment by lseeman** [comment]
Things have changed in the  web authspec is headed for CR, a final wide review draft has been published and more browsers have implention

**Comment by alastc** [comment]
That is good, but doesn’t answer the question about 2 factor, we haven’t established a reasonable method for doing that, even with webauth. 
https://lists.w3.org/Archives/Public/w3c-wai-gl/2017OctDec/0722.html

We also have the question about using password managers, which should change how the SC works: https://lists.w3.org/Archives/Public/w3c-wai-gl/2017OctDec/0825.html

**Comment by lseeman** [comment]
Hi
Nist is not recommending using the 2 step authentication via coping a code from sms. you need to use a token or app that is more secure. Anyway,if you allow an alternative to the two step authentication with copying , you are fine.

We also said that  using password managers will be fine if we write a technique that ensures both the level of security and does not itself require a password. We can simply write that as a techneque

**Comment by alastc** [comment]
HI Lisa,

Agreed about SMS, I've been saying we shouldn't recommend SMS from the start, but the popular 2FA method is Google Auth/Authy, which is the app with 6 digits that change every 30 seconds.

> Anyway,if you allow an alternative to the two step authentication with copying , you are fine.

I don't understand what you mean, you either have 2FA or you don't? It is possible (and more usable) to skip the second factor for re-authentication, but the current wording is for authentication (which I assume means every time). Or you might have multiple methods, but adding methods means more cost, so the effect of the SC will be to mandate particular methods.

**Comment by awkawk** [comment]
Official WG response: Thank you for your comment. This SC has been removed from the draft and will be able to be considered in a future revision.
